engine/src/flutter/sky/tools/create_ios_framework.py

```diff
@@ -30,6 +30,7 @@ def main():
   parser.add_argument('--simulator-arm64-out-dir', type=str, required=False)
   parser.add_argument('--strip', action='store_true', default=False)
   parser.add_argument('--dsym', action='store_true', default=False)
+  parser.add_argument('--skip-extension-safe-frameworks', action='store_true', default=False)
```

Comment:
To avoid the double negatives below (if not skip), I think you can make this:
```
parser.add_argument('--skip-extension-safe-frameworks', dest='include_extension_safe_frameworks', action='store_false', default=True)
```
Then instead of `if not args.skip`, you'll have `if args.include`

Also, I think we'd normally say `--no-` rather than `--skip-`.

Reply:
Oh thank you! Done.